db/db_impl/db_impl_open.cc

```diff
@@ -255,6 +255,25 @@ Status DBImpl::ValidateOptions(const DBOptions& db_options) {
   return Status::OK();
 }
 
+#ifdef OS_LINUX
+std::unordered_set<std::string> DbPaths(
+    const std::string& dbname,
+    const DBOptions& opt,
+    const std::vector<ColumnFamilyDescriptor>& cfds) {
+  std::unordered_set<std::string> paths;
+  paths.insert(dbname);
+  for (const auto& p : opt.db_paths) {
+    paths.insert(p.path);
+  }
+  for (const auto& cfd : cfds) {
+    for (const auto& p : cfd.options.cf_paths) {
+      paths.insert(p.path);
+    }
+  }
+  return paths;
+}
```

Comment:
Do we handle DB::CreateColumnFamily() case somewhere?

Reply:
added logic in `DBImpl::CreateColumnFamilyImpl`